ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -226,6 +226,7 @@ func loadBalancerFlags() []cli.Flag {
 	containerNameUsageString := fmt.Sprintf("[Optional] Specifies the container name (as it appears in a container definition). This parameter is required if --%s or --%s is specified.", flags.LoadBalancerNameFlag, flags.TargetGroupArnFlag)
```

Comment:
Don't forget [deprecated] in the usage for old flags :)

Reply:
So I just put [deprecated] before the [Optional]?

Reply:
```
		cli.StringFlag{
			Name: flags.ComposeServiceNamePrefixFlag,
			Usage: fmt.Sprintf(
				"[Deprecated] Specifies the prefix added to an ECS service created from a compose file. Format <prefix><project-name>. (defaults to empty)",
			),
		},
```

Maybe [Deprecated] instead of [Optional], since I think deprecated field cannot be necessary.

Reply:
Ah I just sent another PR out. Will do when include the change when I also add the unit tests